### PR TITLE
projects page, and front page

### PR DESCRIPTION
**Pull request by Daniel-Awai**

### CODE REVIEW

**Comment by Daniel-Awai** [comment]
Added bg options and projects stuff

**Comment by Daniel-Awai** [comment]
jake jake jake

File: _projects/template.markdown [hunk 1]
```diff
@@ -4,4 +4,4 @@ contributors: "Jake Loo" # Contributors, Contributors
 link: "http://google.com" # Link to github..?
 ---
 #Description
```
Comment (JakeLoo) [inline _projects/template.markdown]: You don't need #Description here I guess?

**Comment by JakeLoo** [comment]
:laughing: @Daniel-Awai  self-assign.

**Comment by JakeLoo** [comment]
:shipit: :rocket: :package:

**Pull request merged**